_includes/nav-button.html

```diff
@@ -0,0 +1 @@
+<a href="#nav">Contents &darr;</a>
```

Comment:
Sorry, yes, this wasn't quite finished. :blush

- Setting this symbol in the Sass: yes! Will do. :)
- My feeling is that the word `Contents` here (and in the `h2`s of `nav.html` should be in `yml` somewhere because it feels like part of the content, not the presentation. For the word in the nav, especially, it makes me feel a bit funky to have an action-based thing (the link) coming from CSS instead of the HTML.